added missing keys to ja_jp file

graycat27 opened the pull request:
based on en_us.lang

added some missing keys only that which I could check how these options works.

graycat27 committed:
Updated ja-jp file

based on en_us.lang
キーの追加対象は、設定の影響内容を操作して確認できたものに限る

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/thexaero/Better-PVP-Translations?pullRequest=1) <br/>All committers have signed the CLA.

thexaero commented:
Thanks for the contribution!
You did a great job. The lines that you added seem to mean exactly what they are supposed to even after translating them back with Google Translate. This isn't always the case, so that's awesome.
Although I have noticed that you have translated some tooltips for options that don't have a translation for their name. For example, _"Entity Name Tags"_. And I also found one where the actual option values are translated but not the option name: _"Main Entity As"_. Might be other ones too. Seems a bit weird. Not sure if you're aware of that. Also, some tooltips are REALLY long in width because they contain no spaces. They still fit the screen but it looks a bit strange. I assume it's because spaces are not used in Japanese but you can still use them here for line breaks, as done by the previous translators.

We can leave it like that for now but, if you could fix this stuff, I would definitely appreciate it! Would probably be a good idea to add the untranslated English lines in between, for future translators, but I can do that myself, so no worries.
I wonder if you weren't able to test the file because of it being a .lang and not a .json? I still support older MC versions and I made myself a .lang to .json converter a while back to port the mods. This is why the repo contains .lang files. You can test the file on 1.12.2. Perhaps I should make a .json to .lang converter too, and share it.

graycat27 committed:
update ja_jp more

added some more missing keys which are the option names or choices.

fixed JP expression at L242 and L270(in new file line num.)
fixed white-space to show better in tooltips at L260 and L271 (in new file line num.)

graycat27 commented:
Thanks for giving me a comment.


> I wonder if you weren't able to test the file because of it being a .lang and not a .json?

Thanks to the hint that langs are stored with json, I was able to check the display of the Japanese which I added.
I checked the latest 1.16.1 forge mod jar file. I replaced the json file with the one I edited with 7-zip.

So, I understand what you said as... 
> some tooltips are REALLY long in width because they contain no spaces.

It's because as you guessed, Japanese doesn't need white-spaces.
I inserted some white-spaces in a long one.


> Would probably be a good idea to add the untranslated English lines in between, for future translators, but I can do that myself, so no worries.

I think its a good idea! In this commit, I didn't do this for now, but it may helps future translators.

graycat27 reviewed (commented):
L218 gui.xaero_cave_zoom should be "洞窟でのズーム倍率"
L231 still over flow the button width
L247 still too long in a line. need get better

thexaero commented on ja_jp.lang:
```diff
@@ -239,34 +239,40 @@ gui.xaero_show_light_level=足元の明るさの表示
 gui.xaero_24h=24h
 gui.xaero_12h=12h
-gui.xaero_temporary=仮
+gui.xaero_temporary=仮削除
```
This translates back as "Temporary Deletion" for me but it should just be "Temporary". It's not caused by deletion only. There is no such thing as temporary deletion in the mod.

graycat27 replied:
> It's not caused by deletion only.

Sorry I missed the way to make the waypoints Temporary stats.
I found, adding with [+]key will make the waypoints Temporary. 
So, your comment is right. Its translate should "仮", not "仮削除".

thexaero replied:
Alright!

thexaero commented on ja_jp.lang:
```diff
@@ -239,34 +239,40 @@ gui.xaero_show_light_level=足元の明るさの表示
+gui.xaero_waypoints_edit_mode_individually=個別
+gui.xaero_waypoints_edit_mode_all=全件
 gui.xaero_compass=方角
+gui.xaero_main_entity_as=プレイヤーアイコン
```
Does this specify that it's the main player, not just any player?

graycat27 replied:
I see in this translation, users may think this option will change all players icon...
I suggest "自身のアイコン" for this.

thexaero replied:
That sounds good to me.

thexaero commented on ja_jp.lang:
```diff
@@ -239,34 +239,40 @@ gui.xaero_show_light_level=足元の明るさの表示
 gui.xaero_box_cave_zoom=洞窟マップモードに切り替わった時の表示ズーム倍率
 gui.xaero_box_entity_depth=高度差が大きい時にエンティティのアイコンを暗く表示します
 gui.xaero_box_height_limit=エンティティを表示する高度差の制限値を設定します
+gui.xaero_entity_nametags=エンティティ名の表示
```
Does this mean name tag or name? Should be name tag.

graycat27 replied:
"エンティティ名" may be interpreted as displaying like "Cow", "Pig" or "Blaze", not sure for name TAG...
This is same meaning with tooltip key
`gui.xaero_box_entity_nametags=名づけられたエンティティの名前...`
So, I`ll fix these two for same expression.

thexaero replied:
The tooltip says something like "named entities", so I thought it was fine. But yeah, check what the "name tag" item is called in Japanese. Might work here too.

thexaero reviewed (commented):
Thanks! Great job.
I'm not sure if some of the lines are translated correctly, so let me know.

graycat27 commented:
thanks for your review.
I'll fix these tomorrow (≒ in about 20h later), its 11PM here so i'm going to sleep tonight.

thexaero commented:
Alright, I appreciate it! No hurry at all.

graycat27 committed:
fixed for review

self review: except for the last one, no need to fix
other review: fixed them

graycat27 commented:
checked with the latest forge 1.16.1 jar (20.20.1)
I didn't reflect my self review
>L218 gui.xaero_cave_zoom should be "洞窟でのズーム倍率"
>L231 still over flow the button width

because, L218's option is able to set "OFF".
and some other options are also overflow, so I thought not necessary to fix now.

thexaero commented:
Good job! Looks good to me right now. Thank you so much for this! Don't worry about the slight overflow. I think I'm going to increase the button width slightly in the future. It doesn't currently work too well for English either. Let me know if you think the PR is ready for merge.

graycat27 commented:
I think this PR is ready for merge :)

thexaero commented:
Alright! Thank you. I really appreciate it. I'm probably going to include this in the following update after 20.20.2, since that one is already out for patrons.
